packages/block-editor/src/components/dimension-control/index.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import { isFunction } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	Icon,
+	SelectControl,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+import {
+	Fragment,
+} from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import sizesTable, { findSizeBySlug } from './sizes';
+
+export function DimensionControl( props ) {
```

Comment:
Why not add this to the components folder?
It seems generic and has no dependencies at all on the block editor?

Reply:
Probably a good point. I'm trying to recall why I originally added this to the Block Editor components. Could be a hangover of having decoupled this component from a larger PR.

If you're recommending moving it I'm happy to go with that.

Reply:
I think the component is in the right place now 👍